pulpanalytics/tests/test_summarize.py

```diff
@@ -67,3 +69,24 @@ def test_summary_worker_count(monkeypatch, db, persistent_min_age_days):
         assert daily_summary.summary.online_workers == Summary.OnlineWorkers(
             processes__avg=2, hosts__avg=1
         )
+
+
+def test_summary_postgresql_version(monkeypatch, db):
+    assert not DailySummary.objects.exists()
+    yesterday = timezone.now() - timezone.timedelta(days=1)
+    with monkeypatch.context() as mp:
+        mp.setattr(timezone, "now", lambda: yesterday)
+        System.objects.create(system_id=uuid.uuid4(), postgresql_version=0)
+        System.objects.create(system_id=uuid.uuid4(), postgresql_version=90200)
+        System.objects.create(system_id=uuid.uuid4(), postgresql_version=90200)
+        System.objects.create(system_id=uuid.uuid4(), postgresql_version=100001)
+
+    call_command("summarize")
+    daily_summary = DailySummary.objects.order_by("date").last()
+    assert daily_summary
+    expected_summary = [
+        {"version": 0, "count": 1},
+        {"version": 90200, "count": 2},
+        {"version": 100001, "count": 1},
+    ]
```

Comment:
You may need to compare to actual protobuf objects, or you serialize `postgresql_version` in to json.

Reply:
I pushed a version that does that but it still doesn't work. I'm adding one more test also, I'll fix this one too, and then I think a merge would be ok. Thanks for all the collab!